verticaltabs.js

```diff
@@ -910,10 +915,6 @@ VerticalTabs.prototype = {
   },
 
   resizeTabs: function () {
-    if (this.resizeTimeout > 0) {
```

Comment:
because it calls `  this.actuallyResizeTabs();` and inside `actuallyResizeTabs` is the exact same code.
<img width="403" alt="screen shot 2016-11-14 at 9 34 55 am" src="https://cloud.githubusercontent.com/assets/10803178/20268556/a69c9eca-aa4d-11e6-87a9-7cca05f964ed.png">